### PR TITLE
[v1 docs] app extension discover section

### PR DESCRIPTION
**Pull request by lucasfernog**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested with all Quasar themes
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [ ] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar-framework.org/tree/dev/source) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by hawkeye64** [comment]
@lucasfernog Nice PR. I think this is a task that should be done before docs server is built and it creates consumable JSON. This way, internet NPM searching is not by per-person when the page is viewed. It also gets rid of all the XSS vectors. It'd also be nice, if possoible, to have Official Quasar app-exts listed in a top section (@quasar/quasar-app-extension-XXX).
To update the JSON, just rerun the task any time.

**Comment by nothingismagick** [comment]
Going into a bit more detail about what @hawkeye64 said:

- First of all, reliance on 3rd party site known to have reliability issues is a no-go. Introducing any kind of AXIOS call is a technical XSS threat, and you are directly injecting the response (unvalidated unless I am mistaken) into the vue object. Of course its a stretch to think that someone would waste their zero-days on hacking NPM in order to incapacitate quasar.
- Next, while it MIGHT be a good idea to list all extensions out there, my personal feeling here is that we cannot edit this list in case an extension goes rogue.

**Comment by nothingismagick** [comment]
Also, I think that we, the maintainers MUST have the ability to blacklist.

File: docs/src/pages/app-extensions/discover.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Discover App Extensions
+---
+
+Here you can search for app extensions you can add to your project.
```
Comment (nothingismagick) [inline docs/src/pages/app-extensions/discover.md]: Here you can search for app extensions to add to your project.

**Comment by rstoenescu** [comment]
@lucasfernog Please leave this as-is for the moment. Thank you for taking time for it.

**Review by nothingismagick — CHANGES_REQUESTED** [review]
We'd like to confer with you and discuss a revision of the process.

**Comment by rstoenescu** [comment]
@lucasfernog Thank you for taking time to contribute. Gonna polish this a bit and it will be in next website deployment.

**Comment by lucasfernog** [comment]
Please let me know if I can help with anything.

**Pull request merged**